builtin/files/userdata/cloud-config-controller

```diff
@@ -4033,6 +4034,20 @@ write_files:
                         operator: In
                         values:
                         - "master"
+                {{- end }}
+                {{- if .KubeDns.AntiAffinityAvailabilityZone }}
+                affinity:
```

Comment:
This looks as though `affinity` is going to be repeated twice at two levels...

```
affinity:
  nodeAffinity:
  affinity:
    podAntiAffinity
```

Reply:
This issue does not seem to occur on your v0.13.x and v0.14.x branches! 👍

Reply:
Good eyes, fixed!